brewtils/schemas.py

```diff
@@ -440,6 +440,7 @@ class OperationSchema(BaseSchema):
         "RequestTemplate": RequestTemplateSchema,
         "Role": RoleSchema,
         "System": SystemSchema,
+        "Operation": OperationSchema,
```

Comment:
Was this not in there already?? How did our tests work?? :facepalm:

Let's chat about this... was it necessary for this PR?